src/helpers/linkQueries.ts

```diff
@@ -0,0 +1,110 @@
+import Fuse from "fuse.js";
+import { Pool } from 'pg';
+import { Config } from "../config";
+import { logger } from "../logger";
+
+export interface Link {
+  id: number,
+  shorten_link: string,
+  url: string,
+}
+
+var localLinks: Link[], pool: Pool;
+
+const ConnectToDB = async () => {
+  try {
+    pool = new Pool({
+      host: Config?.db_host,
+      user: Config?.db_user,
+      database: Config?.db_name,
+      password: Config?.db_password,
+      port: Config?.db_port
+    });
+    await pool.connect();
+    await SeedingData()
+  } catch (err) {
+    logger.debug(err);
+  }
+};
+
+const SeedingData = async () => {
+  try {
+    await pool.query('CREATE TABLE IF NOT EXISTS links (id bigserial primary key, shorten_link text, url text);')
```

Comment:
why have `bigserial` as a primary key? isnt `shorten_link` the key in the link key value store? 

I think it would be faster to have no id and use `shorten_link` as primary key. That way we would be searching using primary keys instead of a value in the table.

Reply:
I used `id` as a key since I thought we will also need to update the `shorten_link`